core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -59,6 +67,46 @@ public interface Configs {
           .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
           .createWithDefault(DEFAULT_ENTITY_RELATIONAL_STORE);
 
+  ConfigEntry<String> ENTITY_RELATIONAL_JDBC_BACKEND_TYPE =
+      new ConfigBuilder(ENTITY_RELATIONAL_JDBC_BACKEND_TYPE_KEY)
+          .doc("Database type of `JDBCBackend`")
+          .version("0.5.0")
+          .stringConf()
+          .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
+          .createWithDefault(DEFAULT_ENTITY_RELATIONAL_JDBC_BACKEND_TYPE);
+
+  ConfigEntry<String> ENTITY_RELATIONAL_JDBC_BACKEND_URL =
+      new ConfigBuilder(ENTITY_RELATIONAL_JDBC_BACKEND_URL_KEY)
+          .doc("Connection URL of `JDBCBackend`")
+          .version("0.5.0")
+          .stringConf()
+          .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
+          .create();
+
+  ConfigEntry<String> ENTITY_RELATIONAL_JDBC_BACKEND_DRIVER =
+      new ConfigBuilder(ENTITY_RELATIONAL_JDBC_BACKEND_DRIVER_KEY)
+          .doc("Driver Name of `JDBCBackend`")
+          .version("0.5.0")
```

Comment:
use `ConfigConstants.VERSION_0_5_0` to instead the string

Reply:
Done